src/pages/experiments/[experimentId]/data-processing/index.jsx

```diff
@@ -391,16 +391,20 @@ const DataProcessingPage = ({ experimentId, experimentData, route }) => {
             <Col>
               {steps[stepIdx].multiSample && (
                 <Button
+                  disabled={processingConfig[steps[stepIdx].key]?.preFiltered}
```

Comment:
Shouldn't we do some conditional rendering, either telling the user that the step is disabled because some sample is pre-filtered, or letting him enable/disable as usual? Maybe a onhover text in the pre-filtered case?

Reply:
Hmmm.. I don't know about this. I can add the information in the pop up bar.

Reply:
Added this in commit 02e2653.